packages/flutter/lib/src/material/bottom_app_bar.dart

```diff
@@ -33,6 +33,174 @@ import 'theme.dart';
 /// ```
 /// {@end-tool}
 ///
+/// {@tool dartpad --template=freeform}
+///
```

Comment:
Something like this would be fine:
```suggestion
/// This example shows the [BottomAppBar], which can be configured to have a notch using the
/// [BottomAppBar.shape] property. This also includes an optional [FloatingActionButton], which illustrates
/// the [FloatingActionButtonLocation]s in relation to the [BottomAppBar].
```